test/data-source.api.test.ts

```diff
@@ -44,11 +47,21 @@ const generateDataSourceMock = (): DataSourceMock => {
   };
 };
 
+const middlewareMock = (
+  _req: Request,
+  _res: Response,
+  next: NextFunction
+): void => {
+  next();
+};
+
 let app: Application;
 
 before(async () => {
   const messageBroker: MessageBroker = messageBrokerMock;
   const connectionUris = await mongoTestServer.getConnectionString();
+  stub(Keycloak.prototype, 'protect').callsFake((): any => middlewareMock);
```

Comment:
If Keycloak is in an external file, it is possible that you won't need to call prototype.

Reply:
It gets angry if I remove prototype